src/sfdxnode/GetNodeWrapper.ts

```diff
@@ -1,17 +0,0 @@
-import * as path from "path";
```

Comment:
We can probably delete the whole sfdxnode folder

Reply:
not sure if I can, looking at the folder we got 6 files in here